rust/datafusion/src/physical_plan/group_scalar.rs

```diff
@@ -63,6 +64,7 @@ impl TryFrom<&ScalarValue> for GroupByScalar {
             ScalarValue::UInt16(Some(v)) => GroupByScalar::UInt16(*v),
             ScalarValue::UInt32(Some(v)) => GroupByScalar::UInt32(*v),
             ScalarValue::UInt64(Some(v)) => GroupByScalar::UInt64(*v),
+            ScalarValue::TimeMillisecond(Some(v)) => GroupByScalar::TimeMillisecond(*v),
```

Comment:
Should we also add support for `ScalarValue::TimeMicrosecond` and `ScalarValue::TimeNanosecond`?

Reply:
Sure thing, I wasn't sure why they weren't added here....